pkg/plugin/twinmaker/client.go

```diff
@@ -49,6 +51,15 @@ type twinMakerClient struct {
 
 // NewTwinMakerClient provides a twinMakerClient for the session and associated calls
 func NewTwinMakerClient(settings models.TwinMakerDataSourceSetting) (TwinMakerClient, error) {
+	httpClient, err := httpclient.New()
+	if err != nil {
+		return nil, err
+	}
+	transport, err := httpclient.GetTransport()
+	if err != nil {
+		return nil, err
+	}
+	httpClient.Transport = httplogger.NewHTTPLogger("grafana-iot-twinmaker-datasource", transport)
```

Comment:
this adds support for [HAR logging](https://github.com/grafana/grafana-plugin-sdk-go/tree/main/experimental/http_logger#http-traffic-logger), which can be useful when debugging.

copy this to your `grafana.ini` to enable and update the `har_log_path`:
```ini
[plugin.grafana-iot-twinmaker-datasource]
har_log_enabled = true
har_log_path = /home/example/twinmaker.har
```